src/components/CityListScreen.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CityList from './CityList';
+
+function CityListScreen(props) {
```

Comment:
This component is just a wrapper for CityList, I don't think it's necessary. If you just put `CityList` instead of `CityListScreen` when creating navigator, you'll get the same thing. 

I would also advise renaming `CityList` to `CityListScreen` once you delete this file.